### PR TITLE
Custom folders part 11

### PR DESCRIPTION
**Pull request by Screenfeed**
CRON to sync files and DB.

### CODE REVIEW

File: inc/functions/custom-folders.php [hunk 1]
```diff
@@ -597,3 +619,173 @@ function imagify_insert_custom_file( $args = array() ) {
...
+		 */
+		if ( $backup_path ) {
+			// Delete the backup file.
+			$filesystem->delete( $backup_path );
+		}
+
+		// Remove the entry from the database.
+		$file->delete_row();
+
+		return new WP_Error( 'no-file', __( 'The file was missing or its path couldn\'t be retrieved from the database. The entry has been deleted from the database.', 'imagify' ) );
```
Comment (Tabrisrp) [inline inc/functions/custom-folders.php]: I'm the annoying guy about contraction :D

File: inc/functions/custom-folders.php [hunk 2]
```diff
@@ -597,3 +619,173 @@ function imagify_insert_custom_file( $args = array() ) {
...
+		}
+	}
+
+	// Hash + modified.
+	$current_hash = md5_file( $file_path );
+
+	if ( ! $old_data['hash'] ) {
+		$new_data['modified'] = 0;
+	} else {
+		$new_data['modified'] = (int) ! hash_equals( $old_data['hash'], $current_hash );
```
Comment (Tabrisrp) [inline inc/functions/custom-folders.php]: you have a compat for hash_equals? (since it's PHP 5.6)

Comment (Screenfeed) [reply inc/functions/custom-folders.php]: You're right, it was missing. The polyfill appeared in WP only in version 3.9.2.

**Comment by Screenfeed** [comment]
Both fixed :)

**Pull request merged**